src/main/java/cist4830/unomaha/tempo/controllers/GoalController.java

```diff
@@ -31,7 +35,7 @@
 
     @GetMapping
     public String index(Model model) {
-        System.out.println("attempted to hit goals page");
+        LOG.info("Tempo index page requested");
```

Comment:
Are we assuming at this point that goals/index is synonymous with index? If not, should we make it clear that we are referring to the goals page?

Reply:
I think they are synonymous at this point